src/test/java/org/apache/commons/collections4/multimap/AbstractMultiValuedMapTest.java

```diff
@@ -211,6 +218,18 @@ public void resetFull() {
         }
     }
 
+    /**
+     * Return a flag specifying the iteration behaviour of the map.
+     * This is used to change the assertions used by specific tests.
+     * The default implementation returns 0 which indicates ordered iteration behaviour.
+     *
+     * @return the iteration behaviour
+     * @see #UNORDERED
+     */
+    protected int getIterationBehaviour(){
```

Comment:
Add a space: `() {`

This method only applies to the iteration of the values of the multi map when it is tested as a collection. I think the method name can be the same as that used in AbstractCollectionTest. But the property should be reused. So UNORDERED must be made public in AbstractCollectionTest.